casanovo/casanovo.py

```diff
@@ -382,7 +395,76 @@ def setup_model(
     return config, model
 
 
-def _get_model_weights() -> str:
+def _get_weights_from_url(
```

Comment:
**nitpick:** I'd move these new functions under `_get_model_weights`, because that one is used first in the code above.